api/cloud/oc_cloud_apis.c

```diff
@@ -472,8 +491,38 @@ cloud_access_register(oc_endpoint_t *endpoint, const char *auth_provider,
   return oc_do_post();
 }
 
+oc_string_t
+cloud_access_deregister_query(const char *uid, const char *access_token, size_t device)
+{
+  oc_string_t q_uid;
```

Comment:
pls initialization to zero value

Reply:
I don't think it's necessary, `oc_concat_strings` creates a new `oc_string_t` from the second and third argument and saves it the first argument, essentially it's an initializer.